CorsixTH/Lua/entities/humanoids/staff/receptionist.lua

```diff
@@ -40,10 +40,10 @@ function Receptionist:tickDay()
 end
 
 function Receptionist:leaveAnnounce()
-  local announcement_priority = AnnouncementPriority.High
+  local announcement_priority = AnnouncementPriority.Critical
```

Comment:
Not your addition but why is this even a variable? Just shove AnnouncementPriority.Critical into the playAnnouncement call?